components/pricing/PricingComparision.vue

```diff
@@ -12,6 +12,12 @@
           </tr>
         </thead>
         <tbody>
+          <tr>
+            <th scope="row">{{ amount[0] }}</th>
```

Comment:
I think it's better to render th Price directly since amount[0] is little confusing for table header.